modules/katamari/src/test/ts/atomic/api/str/ContainsTest.ts

```diff
@@ -38,7 +38,7 @@ describe('atomic.katamari.api.str.ContainsTest', () => {
   it('A string added to a string (at the start) must be contained within it', () => {
     fc.assert(fc.property(
       fc.string(),
-      fc.string(1, 40),
+      fc.string({ minLength: 1, maxLength: 40 }),
       (str, contents) => {
         const r = contents = str;
```

Comment:
The bot pointed out this has been incorrectly guaranteed to pass, probably since it was created. I think it's unlikely to fail but we should still fix the test

```suggestion
        const r = contents + str;
```